docker/image/image.go

```diff
@@ -16,20 +16,27 @@ import (
 	"github.com/docker/docker/registry"
 	"github.com/docker/engine-api/client"
 	"github.com/docker/engine-api/types"
+	"github.com/docker/libcompose/docker/auth"
 )
 
-func inspectImage(ctx context.Context, client client.APIClient, image string) (types.ImageInspect, error) {
+// InspectImage inspect the specified image (can be a name, an id or a digest)
+// with the specified client.
+func InspectImage(ctx context.Context, client client.ImageAPIClient, image string) (types.ImageInspect, error) {
```

Comment:
Those function might be rename `Inspect`, `Remove` and `Pull` though (making it using like `image.Inspect(…)`, …